### ISSUE TITLE
Support lower cost deployment

### ISSUE THREAD

**Issue by pamelafox** [Azure-Samples/azure-search-openai-javascript#215]
Any chance you could support lower cost deployment, ala https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/deploy_lowcost.md ?
That would help hackers who want to deploy this for the upcoming RAGHack.
It'd require adding azd parameters for free skus.

**Comment by anfibiacreativa** [comment]
We will work on this.

**Comment by sinedied** [comment]
I think the most important step we can implement is to support free tier for AI Search, meaning disabling semantic search when it's enabled (if it still a limitation of free tier?).
We can also add an option to allow scaling the backend to 0 to reduce the ACA costs.

**Comment by pamelafox** [comment]
Yes, the lack of semantic ranker is still a limitation. That'd be great, as AI Search can often be the most pricey in the Pricing calculator for these apps.
